engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql

```diff
@@ -801,9 +801,9 @@ INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_vers
 INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '6.7.3', 'opensuse64Guest', 326, now(), 0);
 
 
--- Add SCO OpenServer 6   as support guest os
-INSERT INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (327, UUID(), 7, 'SCO OpenServer 6', now());
--- SCO OpenServer 6   VMWare guest os mapping
+-- Add OpenSUSE Linux   as support guest os
+INSERT INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (327, UUID(), 7, 'OpenSUSE Linux', now());
+-- OpenSUSE Linux   VMWare guest os mapping
```

Comment:
"opensuseGuest" vmware mapping refers to "SUSE openSUSE (32-bit), so I think its better to keep 32bit information in the display name. 64bit has different name in vmware "opensuse64Guest" so its good to differentiate.

Reply:
I´m a little bit confused, because the behaviour of adding/not adding (32-bit) is inconsistent already in this single db-schema upgrade file - for example:

---

https://github.com/apache/cloudstack/blob/015bb20caaabd82774fa3fb261787ef4be9ee90d/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L593

https://github.com/apache/cloudstack/blob/015bb20caaabd82774fa3fb261787ef4be9ee90d/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L594

---

https://github.com/apache/cloudstack/blob/015bb20caaabd82774fa3fb261787ef4be9ee90d/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L620

https://github.com/apache/cloudstack/blob/015bb20caaabd82774fa3fb261787ef4be9ee90d/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L631

---
and opposite

https://github.com/apache/cloudstack/blob/015bb20caaabd82774fa3fb261787ef4be9ee90d/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L718

https://github.com/apache/cloudstack/blob/015bb20caaabd82774fa3fb261787ef4be9ee90d/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L729

---

https://github.com/apache/cloudstack/blob/015bb20caaabd82774fa3fb261787ef4be9ee90d/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L749

https://github.com/apache/cloudstack/blob/015bb20caaabd82774fa3fb261787ef4be9ee90d/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L760



So, what should I use as template?